src/main/webapp/discussion.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const ENDPOINT = '/discussion';
+const PARAM_LECTURE = 'lecture';
+
+const ELEMENT_DISCUSSION = document.querySelector('#discussion');
+const ELEMENT_POST_TEXTAREA = document.querySelector('#post-textarea');
+
+
+/* exported postAndReload */
```

Comment:
Why are there 2 comments for this?

Reply:
The first comment is used to make the linter happy, otherwise when I use the variable in other files it will think it is undefined and refuse to validate